docs/README.md

```diff
@@ -49,11 +49,13 @@
     * [Assertion API](articles/documentation/test-api/assertions/assertion-api.md)
   * [Obtaining Data from the Client](articles/documentation/test-api/obtaining-data-from-the-client.md)
   * [Waiting for Page Elements to Appear](articles/documentation/test-api/waiting-for-page-elements-to-appear.md)
-  * [HTTP Authentication](articles/documentation/test-api/http-authentication.md)
+  * [Authentication](articles/documentation/test-api/authentication/README.md)
+    * [HTTP Authentication](articles/documentation/test-api/authentication/http-authentication.md)
+    * [User Roles](articles/documentation/test-api/authentication/user-roles.md)
```

Comment:
I'd rather change the order. My intuition here is that people more likely will look up for forms authentication scenarios.